src/main/java/org/prebid/server/events/EventUtil.java

```diff
@@ -39,15 +42,33 @@
 
     private static final String LINE_ITEM_ID_PARAMETER = "l";
 
+    private static final List<String> POSSIBLE_TYPES = List.of(WIN_TYPE, IMP_TYPE, VAST_TYPE);
+    private static final List<String> POSSIBLE_V_TYPES = List.of(EventRequest.VastType.START.toString(),
+            EventRequest.VastType.FIRST_QUARTILE.toString(),
+            EventRequest.VastType.MID_POINT.toString(),
+            EventRequest.VastType.THIRD_QUARTILE.toString(),
+            EventRequest.VastType.COMPLETE.toString());
```

Comment:
What about making this utility function(maybe extract this to EnumUtil?)?:
```
private static <E extends Enum<E>> Set<String> stringValues(Class<E> enumeration) {
        return EnumSet.allOf(enumeration).stream()
                .map(Enum::toString)
                .collect(Collectors.toSet());
    }
```

then we can simplify:
```
private static final Set<String> POSSIBLE_TYPES = stringValues(EventRequest.Type.class);
private static final Set<String> POSSIBLE_V_TYPES = stringValues(EventRequest.VastType.class);
```